orchestration/dap_orchestration/solids.py

```diff
@@ -55,7 +65,7 @@ def base_extract_records(context: AbstractComputeExecutionContext) -> DapSurveyT
     arg_dict = {
         "pullDataDictionaries": "true" if context.solid_config["pull_data_dictionaries"] else "false",
         "outputPrefix": f"{context.resources.refresh_directory}/{context.solid_config['output_prefix']}",
-        "endTime": context.solid_config["end_time"],
+        "endTime": check_date_format(context.solid_config["end_time"]),
```

Comment:
I think you'll also need to add a check_date_format call to the env_extract_records solid as it doesn't use this base.